app/views/external_users/claims/misc_fees/litigators/_misc_fee_fields.html.haml

```diff
@@ -1,25 +1,31 @@
-%fieldset
-  %legend.visuallyhidden
-    = t('.miscellaneous_fee')
+- fee = present(f.object)
```

Comment:
I might have missed it 😰 but I can't find any place where this variable is being used in this template. It that's the case ✂️ ! 😈